presto-array/src/main/java/com/facebook/presto/array/Arrays.java

```diff
@@ -104,9 +106,9 @@ else if (buffer.length < capacity) {
         return buffer;
     }
 
-    public static byte[] ensureCapacity(byte[] buffer, int capacity, ExpansionFactor expansionFactor, ExpansionOption expansionOption, ArrayAllocator allocator)
+    public static byte[] ensureCapacity(byte[] buffer, int capacity, int targetSize, ExpansionFactor expansionFactor, ExpansionOption expansionOption, ArrayAllocator allocator)
```

Comment:
targetSize -> estimatedMaxCapacity for consistency

Reply:
Done. I renamed the BlockEncodingBuffer fields names too. E.g. `valuesBufferTargetSize` --> `estimatedValueBufferMaxCapacity`